dataset/src/test/kotlin/org/jetbrains/kotlinx/dl/dataset/preprocessor/PreprocessingFinalShapeTest.kt

```diff
@@ -0,0 +1,134 @@
+package org.jetbrains.kotlinx.dl.dataset.preprocessor
+
+import org.jetbrains.kotlinx.dl.dataset.image.ColorOrder
+import org.jetbrains.kotlinx.dl.dataset.preprocessor.image.*
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.io.File
+
+class PreprocessingFinalShapeTest {
```

Comment:
What do you think about adding tests for comparing preprocessed images with earlier preprocessed and saved to the resources? Is it too difficult to add?